tests/jballerina-unit-test/src/test/resources/test-src/types/typereftype/type_reference.bal

```diff
@@ -355,6 +355,67 @@ function testUnionTypeRefWithMap() {
     assertEqual(map3.toString(), "{\"c\":3}");
 }
 
+public class Student {
```

Comment:
Add few negative test cases to check whether the errors has changed?

Reply:
Updated 601419e96ce753db7d28cdeaca9a809fc8935162